TicketManagement/src/main/java/com/endava/ticketmanagement/service/mapper/OrderToOrderDtoMapper.java

```diff
@@ -0,0 +1,9 @@
+package com.endava.TicketManagement.service.mapper;
+
+import com.endava.TicketManagement.repository.model.Order;
+import com.endava.TicketManagement.service.dto.OrderDto;
+public class OrderToOrderDtoMapper {
+    public static OrderDto converter(Order order){
+        return  new OrderDto();
```

Comment:
Aici si la cealalt DTO mapper sa mappezi datele de care vei avea nevoie si in UI (structura arata ok)